test/pageObjects/webdriver-components/dropdown.js

```diff
@@ -18,7 +18,7 @@ class Dropdown {
   }
 
   //private
-  async _getOptionElements(){
+  async getOptionElements(){
```

Comment:
Why private in a comment? Underscore in a method is a private in JS by convention